data/migrations/24.lua

```diff
@@ -1,3 +1,5 @@
 function onUpdateDatabase()
-	return false
-end
+	print("> Updating database to version 25 (Optimize player direction 4 -> 1 byte)")
```

Comment:
It says version 25 here but this is migration 24.

Reply:
This is just the convention style @marksamman did. I just follow that. Check literally every other migration, they follow the same style. The schema is tagged as version 24, version 25 just returns false until someone builds that migration.